src/components/results/SingleResult.vue

```diff
@@ -1,116 +1,252 @@
+            :src="singleResult.parent.avatarUrl"
+            width="200"
+            height="200"
+            alt="candidateAvatar"
+            data-bs-toggle="tooltip"
+            data-bs-placement="left"
+            title="Go to candidate`s page"
+          />
+        </router-link>
       </div>
-      <div class="col-12 text-center text-md-start">
-        <h3>{{ result.parent.position }}</h3>
+      <div class="col-md-8 col-lg-4 mt-3 mt-lg-2">
+        <h2 class="text-primary text-center text-md-start">
+          {{ singleResult.parent.username }}
+        </h2>
+        <div class="text-center text-md-start fs-4">
+          <font-awesome-icon
+            icon="fa-solid fa-user"
+            class="text-primary"
+          />
+          {{ singleResult.parent.position }}
+        </div>
+        <div class="text-center text-md-start fs-4">
+          <font-awesome-icon
+            icon="fa-regular fa-clock"
+            class="text-primary fs-4"
+          />
+          {{ calculateTime(singleResult.startedAt, singleResult.endedAt) }}
+          minutes
+        </div>
+        <div class="text-center text-md-start fs-4">
+          <font-awesome-icon
+            icon="fa-solid fa-calendar-days"
+            class="text-primary fs-5"
+          />
+          {{ formattingDate(singleResult.endedAt) }},
+          {{ formattingHours(singleResult.endedAt) }}
+        </div>
+        <div class="text-center text-md-start mb-3 fs-4">
+          Result: {{ resultPercentage }}
+        </div>
+        <div class="progress">
+          <div
+            class="progress-bar progress-bar-striped progress-bar-animated"
+            role="progressbar"
+            aria-label="candidate score"
+            :style="`width: ${resultPercentage}`"
+            aria-valuenow="75"
+            aria-valuemin="0"
+            aria-valuemax="100"
+          ></div>
+        </div>
       </div>
-      <div class="col-12 text-center text-md-start">
-        <h4>Result: {{ resultPercentage }}%</h4>
+    </div>
+    <div class="col-12 mt-md-5">
+      <div
+        class="text-center text-md-start shadow border border-secondary p-3 rounded border-1 mt-3"
+      >
+        <h4 class="fw-light">
+          <span class="fw-bolder">Short summary:</span>
+          {{ singleResult.parent.feedback }}
+        </h4>
       </div>
-      <CategoryListItem
-        v-for="(category, index) in categories"
-        :key="category"
-        :item-id="index"
-        :category="category"
-        :questions-array="result.questionAnswer"
-      />
-
-      <div class="col-12 text-center text-md-end mt-3">
-        Start Date: {{ formattingDate(result.startedAt) }}
+      <h4 class="text-center text-md-start ps-md-1 mt-4">Skills</h4>
+      <div class="row">
+        <radial-progress-bar
+          v-for="category in categories"
+          :key="category"
+          class="col-3 mx-4 my-3"
+          :completed-steps="answerPointsByCategory(category)"
```

Comment:
you use this function makes two loops through arrays and it is used 3 times for each category. It's more performant to create a computed for categories points. + the same for `pointsByCategory`